integration-tests/change-base-dir/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+quarkus.openapi-generator.codegen.spec.petstore_json.base-package=org.acme.petstore
+# This one is passed through the Classpath source provider. Uncomment to overwrite it
+#quarkus.openapi-generator.codegen.spec.subtraction_yaml.base-package=org.acme.subtraction
+quarkus.oidc-client.client-enabled=false
+# ${keycloak.url} is provided by test-utils
+# petstore_auth is the name of the security scheme from the openapi spec file (petstore.json)
+# you can configure your OAuth2 Client the way it fits your use case here.
+# see https://quarkus.io/guides/security-openid-connect-client
+quarkus.oidc-client.petstore_auth.auth-server-url=${keycloak.url}
+quarkus.oidc-client.petstore_auth.discovery-enabled=false
+quarkus.oidc-client.petstore_auth.token-path=/tokens
+quarkus.oidc-client.petstore_auth.credentials.secret=secret
+quarkus.oidc-client.petstore_auth.grant.type=password
+quarkus.oidc-client.petstore_auth.grant-options.password.username=alice
+quarkus.oidc-client.petstore_auth.grant-options.password.password=alice
+quarkus.oidc-client.petstore_auth.client-id=petstore-app
+
+quarkus.openapi-generator.codegen.inputBaseDir=openapi-definitions
+quarkus.openapi-generator.codegen.inputFiles=petstore.json
```

Comment:
What is this property for?

Reply:
removed after taking the include testing to include-exclude